libfreerdp/core/input.c

```diff
@@ -967,6 +999,9 @@ BOOL freerdp_input_send_extended_mouse_event(rdpInput* input, UINT16 flags, UINT
 	if (freerdp_settings_get_bool(input->context->settings, FreeRDP_SuspendInput))
 		return TRUE;
 
+	input_update_last_event(input,
```

Comment:
wrong flags, no need to check them here anyway as extended mouse event only has normal button presses